core/kazoo_number_manager/src/knm_phone_number.erl

```diff
@@ -700,8 +701,7 @@ set_module_name(N0, Name, IsBillable)
     N = set_module_name(N0, Name),
     N#knm_phone_number{is_billable = IsBillable};
 set_module_name(N0, Name, 'undefined') ->
-    N = set_module_name(N0, Name),
-    N#knm_phone_number{is_billable = knm_carriers:is_number_billable(N)}.
```

Comment:
Why not? it same as the last `set_module_name/2`,  but setting `is_billable` twice